drafts/str4d-overwinter-sighash/draft1.rst

```diff
@@ -0,0 +1,312 @@
+::
+
+  ZIP: 143
+  Title: Transaction Signature Verification for Overwinter
+  Author: Jack Grigg <jack@z.cash>
+          Daira Hopwood <daira@z.cash>
+  Credits: Johnson Lau <jl2012@xbt.hk>
```

Comment:
We should also credit BitcoinABC because they inspired ticket #2584 by writing up a spec https://github.com/Bitcoin-UAHF/spec/blob/master/uahf-technical-spec.md and https://github.com/Bitcoin-ABC/bitcoin-abc/blob/master/doc/abc/replay-protected-sighash.md demonstrating how to extract the relevant bits from BIP 143 etc.